keyboards/smallice/readme.md

```diff
@@ -0,0 +1,15 @@
+# smAllice
+
+![smAllice](https://i.imgur.com/HAiJnnK.png)
+
+A 40% alice-like keyboard with an arrow cluster.
+
+* Keyboard Maintainer: [Brandon Beltran](https://github.com/armastardo)
+* Hardware Supported: smAllice PCB
+* Hardware Availability: [KeyHive](http://keyhive.xyz/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Please add instructions on how to jump into the bootloader, to flash new firmware.